src/platform/datapath_epoll.c

```diff
@@ -2171,7 +2179,7 @@ CxPlatSocketSendInternal(
         }
     }
 
-    SentByteCount = sendmsg(SocketContext->SocketFd, &Mhdr, 0);
+    SentByteCount = sendmmsg(SocketContext->SocketFd, Mhdrs, (unsigned int)SendCount, 0);
```

Comment:
So `sendmmsg` takes individual message headers for each packet, and not just one for all?

Reply:
Yes. Sendmmsg and recvmmsg are just fast wrappers that can do multiple sendmsg or recvmsg in 1 system call. Once in the kernel, each message is treated completely separately.